pb/ctrl_pb/ctrl.proto

```diff
@@ -39,6 +40,7 @@ message BindRequest {
     BindType bindType = 1;
     string token = 2;
     string serviceId = 3;
+    map<uint32, bytes> hostData = 4;
```

Comment:
Also, we need to make sure we update the `Decoder` component `pb/ctrl_pb/decoder.go`... otherwise all of this new data will be lost when tracing.

Reply:
added tracing of BindRequest,BindResponse, and peerData ids